### DISCUSSION TITLE
Benefits Availability

### DISCUSSION THREAD

**Discussion by RaulJDlCRUZ** [community/community#141092] [GitHub Education] [answered]
### Hi everyone,

Hello everyone! This is my first post on GitHub Education, and I'm looking for help, or at least an explanation of my problem, in order to learn more about how the platform works. I was recently approved to apply for GitHub Education benefits, but I'm still waiting for the availability of the benefits.

![Approved](https://github.com/user-attachments/assets/6c4c2ffd-13f0-4cec-b846-68bf638df01a)

## Before the first rejection

- 
- I set my main email address as my academic email address, and, for my repos and projects, this was not a problem
- The image I sent was an extract of the letter of payment for the 4th year of Computer Engineering (2024/25), so, as it was similar to an invitation letter, it was rejected.

## After the first rejection

- I'm using Firefox as my primary browser, but I used Microsoft Edge, and had no problems.
- I set up two-factor authentication, with GitHub Mobile and Google Authenticator. 
- My public profile was -more- customised/personalized, with a README.md and some interests.
- In the second attempt, the image is a photograph with a mobile phone towards a printed document (non-digital), which is an academic certificate, requested at the secretary's office of the faculty, which certifies that I am a student in that faculty in a specific degree (Computer Engineering). The date of the digital signature of this document is 19 September, so that, with the course having started, it is determined that I am still studying this degree at the faculty.

Now, on the benefit claim page, the following message appears in yellow:

**Hi, RaulJDlCRUZ! You were last verified as a on Oct 3, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.**

And:

**You have verified the email address Raul.Jimenez8@alu.uclm.es on your GitHub account. That academic domain is associated with the school University of Castilla-La Mancha.**

## What's next?

As you can see in the embedded image, the application was approved as of 3 October, however, the GitHub Education benefits were not activated for me, which would take about 3 days, and, if not from 5 days, to contact the community (Source: https://github.com/orgs/community/discussions/111352).
Therefore, the reason for publication is in case the administration needs to make changes or if I am missing any steps or formalities on my side. Thank you very much for your time.

**Comment by davevad93** [comment, marked as answer]
Hi @RaulJDlCRUZ ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @RaulJDlCRUZ was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by RaulJDlCRUZ** [reply]
Hi @davevad93 ,
Hilariously, five minutes after sending the message, I received an email from GitHub Education informing me that I was now part of the GitHub Global Campus, and indeed, in the status I already have a purple label along with:

`Congrats! Your request for @RaulJDlCRUZ was approved and your benefits are now available.`

So, in my case it was a matter of time, maybe the volume of requests, as you said. Thank you very much for your time!
